### DISCUSSION TITLE
My minidexED 42HP rack

### DISCUSSION THREAD

**Discussion by DutchBigEd** [probonopd/MiniDexed#428] [Show and tell]
![case2](https://user-images.githubusercontent.com/17274454/215270541-fbe3d678-cc26-4d2c-8a2d-bffb70a66f6b.jpg)

The electronics is PM, the Raspberry PI 4b is working with the display, rotary encoder and audio-hat.
The pushbuttons, rotary encoders and volume potmeters will be converted to midi with an Arduino Uno and sent to the Pi4b.
The Uno is programmed with HiDuino to act as a native midi device.
http://www.dimitridiakopoulos.com/hiduino 
From left to right:
A 12HP module which is mostly connected directly to the PI4b.
- a uSD connector with an extension cable so I can change the uSD card content without opening the case.
- next to this connector is an Oled display, 128x64
- below this connector is a master volume potmeter followed by a 2x amplifier/buffer
- lft and rgt connectors (left and right audio out, is buffered and amplified 2x)
- midi in and midi thru, a jumper on the board makes it possible to change the thru from midi from the PI4b or from the midi in.

Next is a 4HP volUme module, where the position of 8 potmeters is measured with the Arduino Uno and send using midi to the PI4b.

This instance is a 8HP wide module that functions as follows:
- an Oled display that displays which instance is chosen, what bank and what voice is used with this instance
- two pushbuttons to choose the instance the UNO is midi sending to
- RNG is rotary encoder that sets the range of the filter of the chosen instance
- EMP is emphasis of this filter
- PAN is the pan position of the chosen instance
- DET is detune of this instance

Most right module is a 16HP wide euro module that uses the UNO to scan the 32 pushbuttons
 - voice if this button is pressed, the 32 switches will instantly choose a voice of the chosen bank
 - bank if this button is pressed, the 32 switches will choose 1 of the 32 first banks, pressing again will select bank 33 to 64, this can be extended to 128 banks. 
 - sustn sustain on/off
 - S/N off is all sound off or all notes off, later I will determine what is best (maybe both)

**Comment by Banana71** [comment]
Vintage industrial style is all the rage.  I like it very much. And the functionality is just as impressive.
This is how I imagine the instruments of "Flake" (Rammstein, Keys) to be.

**Comment by diyelectromusic** [comment]
Love it.  That looks amazing :)
